packages/belos/src/BelosPseudoBlockGmresSolMgr.hpp

```diff
@@ -735,9 +737,28 @@ setParameters (const Teuchos::RCP<Teuchos::ParameterList>& params)
     }
   }
 
+  // Optional sublist for OrthoManager parameters
+  RCP<ParameterList> orthoParams;
+  std::string tempOrthoType;
+  // the sublist for ortho options is new (2017).
+  // the intent is that the string parameter 'Orthogonalization'
+  // can optionally define a sublist.
+  const bool haveOrthoParamName = params->isParameter ("Orthogonalization");
```

Comment:
@hkthorn Two-argument ParameterList::get() actually adds the parameter to the list with the default value if it's not already there, right?